packages/esm-ward-app/src/ward-patient-workspace/ward-patient.workspace.tsx

```diff
@@ -2,22 +2,24 @@ import React, { useEffect, useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
 import { InlineLoading, InlineNotification } from '@carbon/react';
 import {
+  age,
+  attach,
   type DefaultWorkspaceProps,
   ExtensionSlot,
-  attach,
   getPatientName,
+  type PatientUuid,
   usePatient,
-  age,
 } from '@openmrs/esm-framework';
 import styles from './ward-patient.style.scss';
+import { type WardPatientCardProps } from '../types';
 
 attach('ward-patient-workspace-header-slot', 'patient-vitals-info');
 
-export interface WardPatientWorkspaceProps extends DefaultWorkspaceProps {
-  patientUuid: string;
+export interface WardPatientWorkspaceProps extends DefaultWorkspaceProps, WardPatientCardProps {
```

Comment:
It's better to export types from a separate file, else, sometimes, the components gets circular imports due to type importing.

Reply:
If a sub component needs this type, it will import the type from the parent and parent imports sub component, hence cyclic imports can happen.